tests/ludwig/data/test_cache_util.py

```diff
@@ -92,3 +92,20 @@ def test_proc_col_checksum_consistency():
     for of1, of2 in zip(config1.output_features, config2.output_features):
         assert of1.name == of2.name
         assert of1.proc_column == of2.proc_column
+
+
+def test_proc_col_checksum_consistency_same_preprocessing_different_types():
+    """Tests that proc_col is equal if checksum are equal."""
```

Comment:
Nit: this tests that proc_cols are different when types are different.

Reply:
Ahh co-pilot 🤦  Just fixed.